models.py

```diff
@@ -82,6 +83,30 @@ def __repr__(self):
         """Repr method."""
         return f"EVC({self._id}, {self.name})"
 
+    @property
+    def enabled(self):
+        """Override the enabled method.
+
+        We consider a EVC enabled whether all the current path are enabled.
+
+        Returns:
+            boolean: True if the links are enabled, othewrise False.
+
+        """
+        return all([link.enabled for link in self.current_path])
+
+    @property
+    def active(self):
+        """Override the active method to consider the Links.
```

Comment:
@macartur For an EVC to be active, it should have been deployed (flows pushed to the switch), otherwise, it should be still active as False, because we can have an EVC created, all links in the path are UP, but the creation date hasn't been reached yet. Could you handle this case in the logic of your code? Also, we should probably take into account the return of send_flow_mods to update the active status since the FlowMod might have some issues to be pushed successfully.